### DISCUSSION TITLE
时间组件面板样式错乱

### DISCUSSION THREAD

**Discussion by shopin-beijing** [element-plus/element-plus#8299] [Q&A]
![image](https://user-images.githubusercontent.com/89066956/174019872-9d8d7f6d-50e0-426a-847c-bb76f7402760.png)
这是神马个情况，时间组件咋都成这个鬼样子了～～

**Comment by KimYangOfCat** [comment]
也许，你应该先读一读这篇关于如何提问的[文章](https://github.com/ryanhanwu/How-To-Ask-Questions-The-Smart-Way/blob/main/README-zh_CN.md) 
Maybe you can read this [article](https://github.com/ryanhanwu/How-To-Ask-Questions-The-Smart-Way/blob/main/README-zh_CN.md) about how to take a question firstly!

**Comment by sxzz** [comment]
https://antfu.me/posts/why-reproductions-are-required-zh

**Reply by izhouteng** [reply]
我也遇到过，是文档上 Config Provider 组件的问题。

复现过程：浏览一下文档上 Config Provider 组件，随后浏览其他支持多语言的组件都无法正常显示文字。

原因：`el-config-provider`示例传了一个不完整的`locale`属性，覆盖了默认的语言包，导致无法显示文字。

**Reply by sxzz** [reply]
Got it. We will fix it.
